### ISSUE TITLE
Version 5.0.0 is incompatible with doorkeeper 5.2.0

### ISSUE THREAD

**Issue by anikolaiev** [doorkeeper-gem/doorkeeper-mongodb#39]
Since 5.2.0.rc3 RedirectUriValidator lives under Doorkeeper namespace. 
this is reflected in lib/doorkeeper/orm/active_record/application.rb
```
-   validates :redirect_uri, redirect_uri: true
+   validates :redirect_uri, "doorkeeper/redirect_uri": true
```
but doorkeeper-mongodb/lib/doorkeeper-mongodb/mixins/mongoid/application_mixin.rb still uses old class which does not exist any more
```
validates :redirect_uri, redirect_uri: true
```

So if I try to run console I get this error
```
NameError:
  uninitialized constant Doorkeeper::RedirectUriValidator
# /Users/andrii/.rvm/gems/ruby-2.6.3/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `block in load_missing_constant'
# /Users/andrii/.rvm/gems/ruby-2.6.3/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:8:in `without_bootsnap_cache'
# /Users/andrii/.rvm/gems/ruby-2.6.3/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `rescue in load_missing_constant'
# /Users/andrii/.rvm/gems/ruby-2.6.3/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:58:in `load_missing_constant'
# ./config/initializers/doorkeeper.rb:4:in `<main>'
```

Gemfile
gem 'doorkeeper', '5.2.0'
gem 'doorkeeper-mongodb', '~> 5.0.0'

**Comment by rgaufman** [comment]
Any workaround for this?

**Comment by alesna97** [comment]
same problem here

**Comment by anikolaiev** [comment]
fixed it like this for now

config/initializers/doorkeeper.rb
`require 'doorkeeper/orm/active_record/redirect_uri_validator'
class RedirectUriValidator < Doorkeeper::RedirectUriValidator; end`

**Comment by dfabreguette-ap** [comment]
Yep ! this helped me a lot !
Here's my fork https://github.com/dfabreguette-ap/doorkeeper-mongodb

**Comment by nbulaj** [comment]
Would be added with a new version (5.2) after merge of #42

**Comment by nbulaj** [comment]
Released to rubygems as 5.2.0, feel free to report issues and PR's to fix them :)
Thanks! :tada:
